recipe for target 'binary-local' failed

```
[root@iZrj9emxe3fhdrziho0mthZ skopeo-0.1.34]# cat /etc/redhat-release
CentOS Linux release 7.6.1810 (Core)
```

```
[root@iZrj9emxe3fhdrziho0mthZ skopeo-0.1.34]# make all
docker build  -f Dockerfile.build -t skopeobuildimage .
Sending build context to Docker daemon 14.67 MB
Step 1/4 : FROM ubuntu:17.10
 ---> e211a66937c6
Step 2/4 : RUN apt-get update && apt-get install -y     golang     btrfs-tools     git-core     libdevmapper-dev     libgpgme11-dev     go-md2man     libglib2.0-dev     libostree-dev
 ---> Using cache
 ---> add8c6365bab
Step 3/4 : ENV GOPATH /
 ---> Using cache
 ---> 85edf1bac744
Step 4/4 : WORKDIR /src/github.com/containers/skopeo
 ---> Using cache
 ---> cc41e10a9642
Successfully built cc41e10a9642
docker run --rm --security-opt label=disable -v $(pwd):/src/github.com/containers/skopeo \
	skopeobuildimage make binary-local  BUILDTAGS='btrfs_noversion libdm_no_deferred_remove '
go build "-buildmode=pie" -ldflags "-X main.gitCommit=" -gcflags "" -tags "btrfs_noversion libdm_no_deferred_remove " -o skopeo ./cmd/skopeo
vendor/github.com/klauspost/compress/flate/inflate.go:13:2: cannot find package "math/bits" in any of:
	/src/github.com/containers/skopeo/vendor/math/bits (vendor tree)
	/usr/lib/go-1.8/src/math/bits (from $GOROOT)
	/src/math/bits (from $GOPATH)
Makefile:83: recipe for target 'binary-local' failed
make: *** [binary-local] Error 1
make: *** [binary] Error 2
```

--- a/Dockerfile.build
+++ b/Dockerfile.build
@@ -1,4 +1,4 @@
-FROM ubuntu:17.10
+FROM ubuntu:18.10
 
 RUN apt-get update && apt-get install -y \
     golang \

